### PR TITLE
fix: YoStarKR SkipThePreBattlePlot

### LINKED ISSUE #8644: 대리지휘 중 스토리 장면에서 멈추는 현상
### 在提问之前...

- [X] 我理解 Issue 是用于反馈和解决问题的，而非吐槽评论区，将尽可能提供更多信息帮助问题解决
- [X] 我填写了简短且清晰明确的标题，以便开发者在翻阅 issue 列表时能快速确定大致问题。而不是“一个建议”、“卡住了”等
- [X] 我已查看最新测试版本的更新内容，并未提及该 bug 已被修复的情况
- [X] 我已检查了置顶议题（pinned issue），确认我的问题未被提及

### 问题描述

스테이지 진행 중 스토리 장면이 나올 경우 스토리를 스킵 하지 않고 그대로 매크로가 멈추는 현상이 있습니다. 반면 전투가 끝난 후 진입하는 스토리는 스킵을 잘하는 것으로 보아 오류인 것으로 생각되니 조속한 픽스 바랍니다.

### Version

5.2.0

### 日志文件

모두에게서 일어나는 일인 것 같습니다.

### 截图

_No response_

### 还有别的吗？

_No response_

### PR DESCRIPTION
**Pull request by HX3N**
fix #8644

<details><summary>log</summary>

```log
[2024-03-21 17:46:20.776][TRC][Pxbf48][Txb42c] asst::PipelineAnalyzer::analyze BattleOfficiallyBegin
[2024-03-21 17:46:20.777][TRC][Pxbf48][Txb42c] asst::PipelineAnalyzer::analyze SkipThePreBattlePlot
[2024-03-21 17:46:20.777][INF][Pxbf48][Txb42c] asst::TemplResource::get_templ lazy load SkipThePreBattlePlot.png
[2024-03-21 17:46:20.784][TRC][Pxbf48][Txb42c] match_templ | SkipThePreBattlePlot.png score: 0.993039 rect: [ 1146, 44, 76, 19 ] roi: [ 1100, 0, 180, 100 ]
[2024-03-21 17:46:20.785][INF][Pxbf48][Txb42c] Assistant::append_callback | SubTaskStart {"class":"asst::ProcessTask","details":{"action":"ClickSelf","algorithm":"MatchTemplate","exec_times":1,"max_times":2147483647,"result":{"score":0.993039},"task":"SkipThePreBattlePlot"},"first":["BattleStartAll"],"pre_task":"BattleStartNormal","subtask":"ProcessTask","taskchain":"Copilot","taskid":4,"uuid":"a5abb3448e003947"}
[2024-03-21 17:46:20.785][TRC][Pxbf48][Txb42c] minitouch click: (1180, 51)
[2024-03-21 17:46:20.886][INF][Pxbf48][Txb42c] Assistant::append_callback | SubTaskCompleted {"class":"asst::ProcessTask","details":{"action":"ClickSelf","algorithm":"MatchTemplate","exec_times":1,"max_times":2147483647,"result":{"score":0.993039},"task":"SkipThePreBattlePlot"},"first":["BattleStartAll"],"pre_task":"BattleStartNormal","subtask":"ProcessTask","taskchain":"Copilot","taskid":4,"uuid":"a5abb3448e003947"}
[2024-03-21 17:46:20.886][TRC][Pxbf48][Txb42c] ready to sleep 500
[2024-03-21 17:46:21.386][TRC][Pxbf48][Txb42c] end of sleep 500
```
</details>

### CODE REVIEW

**Comment by HX3N** [comment]
What's the problem

**Comment by status102** [comment]
> What's the problem Merge should be no problem... right?

No. there are some things wrong. You could search `[WRN]` in the log of smoke-testing.
( btw, there are too much lines, annoying thing

**Comment by Constrat** [comment]
Don't tell me it's all because the position is wrong xD, I never noticed .,

**Comment by status102** [comment]
[2024-03-21 09:01:39.450][ERR][Px143c][Tx067c] ����� SkipThePreBattlePlot has unknown key: threshold

File: resource/global/YoStarKR/resource/tasks.json [hunk 1]
```diff
@@ -3490,6 +3490,20 @@
             "하드모드개시"
         ]
     },
+    "SkipThePreBattlePlot": {
+        "threshold": 0.9,
```
Comment (status102) [inline resource/global/YoStarKR/resource/tasks.json:3494]:
```suggestion
        "templThreshold": 0.9,
```

Comment (HX3N) [reply resource/global/YoStarKR/resource/tasks.json:3494]: Oh, darn. I was being stupid.

Comment (HX3N) [reply resource/global/YoStarKR/resource/tasks.json]: Thank you... I really did something stupid...

**Comment by HX3N** [comment]
> Don't tell me it's all because the position is wrong xD, I never noticed .,

If position refers to ROI, then it probably doesn't matter. Modifying the maskRange in KR improved recognition significantly.

**Comment by Constrat** [comment]
Huh... The recognition must be so bad that MAA doesn't even try to recognize, usually it gives a score , but in my case it wasn't even trying

**Comment by status102** [comment]
> Huh... The recognition must be so bad that MAA doesn't even try to recognize, usually it gives a score , but in my case it wasn't even trying

<0.5 won't print trace log

**Comment by Constrat** [comment]
@HX3N confirm only mask range, still works?

**Comment by HX3N** [comment]
Yes, I just checked and it's working fine. Since the existing image in KR is already black except for "SKIP," modifying just the maskRange should suffice.

<details><summary>log</summary>

```log
[2024-03-22 00:10:41.638][INF][Px8aa4][Tx430c] {"class":"asst::ProcessTask","details":{"cur_retry":0,"retry_times":20,"to_be_recognized":["BattleOfficiallyBegin","SkipThePreBattlePlot","Stop"]},"first":["BattleStartAll"],"pre_task":"BattleStartNormal","subtask":"ProcessTask","taskchain":"Copilot","taskid":12}
[2024-03-22 00:10:41.962][INF][Px8aa4][Tx430c] Call ` adb/platform-tools/adb.exe -s 127.0.0.1:5575 exec-out "screencap | gzip -1" ` ret 0 , cost 323 ms , stdout size: 601544 , socket size: 0
[2024-03-22 00:10:41.974][TRC][Px8aa4][Tx430c] asst::PipelineAnalyzer::analyze BattleOfficiallyBegin
[2024-03-22 00:10:41.976][TRC][Px8aa4][Tx430c] asst::PipelineAnalyzer::analyze SkipThePreBattlePlot
[2024-03-22 00:10:41.976][INF][Px8aa4][Tx430c] asst::TemplResource::get_templ lazy load SkipThePreBattlePlot.png
[2024-03-22 00:10:41.989][TRC][Px8aa4][Tx430c] match_templ | SkipThePreBattlePlot.png score: 0.998567 rect: [ 1146, 37, 82, 31 ] roi: [ 1096, 0, 182, 131 ]
[2024-03-22 00:10:41.989][INF][Px8aa4][Tx430c] Assistant::append_callback | SubTaskStart {"class":"asst::ProcessTask","details":{"action":"ClickSelf","algorithm":"MatchTemplate","exec_times":1,"max_times":2147483647,"result":{"score":0.998567},"task":"SkipThePreBattlePlot"},"first":["BattleStartAll"],"pre_task":"BattleStartNormal","subtask":"ProcessTask","taskchain":"Copilot","taskid":12,"uuid":"a5abb3448e003947"}
[2024-03-22 00:10:41.989][TRC][Px8aa4][Tx430c] minitouch click: (1185, 49)
[2024-03-22 00:10:42.090][INF][Px8aa4][Tx430c] Assistant::append_callback | SubTaskCompleted {"class":"asst::ProcessTask","details":{"action":"ClickSelf","algorithm":"MatchTemplate","exec_times":1,"max_times":2147483647,"result":{"score":0.998567},"task":"SkipThePreBattlePlot"},"first":["BattleStartAll"],"pre_task":"BattleStartNormal","subtask":"ProcessTask","taskchain":"Copilot","taskid":12,"uuid":"a5abb3448e003947"}
[2024-03-22 00:10:42.090][TRC][Px8aa4][Tx430c] ready to sleep 500
[2024-03-22 00:10:42.590][TRC][Px8aa4][Tx430c] end of sleep 500
```
</details>

**Comment by Constrat** [comment]
Mask range Is pretty powerful for matching white text. Ignoring all colours except the last 50 whites does wonders. I'll also add it for the en

**Pull request merged**